src/uu/rev/src/rev.rs

```diff
@@ -12,10 +12,21 @@ use uucore::{error::UResult, format_usage, help_about, help_usage};
 const ABOUT: &str = help_about!("rev.md");
 const USAGE: &str = help_usage!("rev.md");
 
+mod options {
+    pub const FILE: &str = "file";
+    pub const ZERO: &str = "zero";
+}
+
 #[uucore::main]
 pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     let matches: clap::ArgMatches = uu_app().try_get_matches_from(args)?;
-    let files = matches.get_many::<String>("file");
+    let files = matches.get_many::<String>(options::FILE);
+    let zero = matches.get_flag(options::ZERO);
+
+    let sep = match zero {
+        false => b'\0',
+        true => b'\n',
+    };
```

Comment:
While this works fine, I think it's more idiomatic to use an `if`/`else` because the `match` expression is a `bool`.

Reply:
fixed